open-metadata-resources/open-metadata-deployment/docker/egeria-uistatic/dist/nginx.conf

```diff
@@ -0,0 +1,15 @@
+# SPDX-License-Identifier: Apache-2.0
+# Copyright Contributors to the Egeria project
+
+server {
+    listen 80;
```

Comment:
should have the HTTPS port also

Reply:
Agreed should look at https. I'll get this in & then let's catch changes in another iteration through? Means anyone using the charts etc in master can be working again whilst we refine. I think that nginx config needs more refinement too